Chemical-Curation/chemcurator_django
# 26 dockerize admin

Closes #26 

Environment variables can now always be set in a ".env" file at the root of your project. See "template.env" for more details.

A _lot_ of Django Cookie Cutter files went away
- when consolidating the `requirements.txt` files and the `settings.py` files
- when implementing the Dockerfile and `docker-compose.yaml` script

To test out the image, run

```shell
docker build . -t chemreg

docker run \
    --rm \
    --network=host \
    -e ALLOWED_HOSTS=localhost,0.0.0.0,127.0.0.1 \
    -e CACHE_URL=locmemcache:// \
    -e DATABASE_URL=postgres://USER:PASSWORD@127.0.0.1:5432/chemreg \
    -e DEBUG=false \
    -e SECRET_KEY=test \
    -e URL_CONF=admin \
    -e WEB_CONCURRENCY=1 \
    chemreg
```

Just make sure to change `DATABASE_URL` to include your user and password.
